### DISCUSSION TITLE
What is the "multi-tenant" feature and how does it effect me when self hosting?

### DISCUSSION THREAD

**Discussion by yousif-khalil-abdulkarim** [vendurehq/vendure#3129] [Q&A] [answered]
Hi everybody, could someone please clarify what "Additional installations (multi-tenant)" means in the pricing section? How does it effect me if I self host.

**Comment by michaelbromley** [comment, marked as answer]
Hi!
That only relates to the commercial license. If you are using the community (GPL) license it is not applicable - you can self host any number of instances.
